src/module/server-instance/ServerInstance.ts

```diff
@@ -0,0 +1,37 @@
+import {App} from "../../ManagementConsole";
+import {ServerInstanceCtrl} from "./ServerInstanceCtrl"
+import {ServerAddress} from "../../services/server/ServerAddress";
+import {ServerService} from "../../services/server/ServerService";
+import {ServerGroupService} from "../../services/server-group/ServerGroupService";
+
+const module: ng.IModule = App.module("managementConsole.server-instance", []);
+
+module.controller("ServerInstanceCtrl", ServerInstanceCtrl);
+
+// @ngInject
+module.config(($stateProvider: ng.ui.IStateProvider) => {
+  $stateProvider.state("server-instance", {
+    parent: "root",
+    url: "server-groups/:serverGroupName/:host/:server",
+    templateUrl: "module/server-instance/view/server-instance.html",
+    controller: ServerInstanceCtrl,
+    controllerAs: "ctrl",
+    params: {
+      serverGroupName: null,
+      host: null,
+      server: null
+    },
+    resolve: {
+      coord: ["$stateParams", "serverGroupService", ($stateParams, serverGroupService) => {
+        var server = new ServerAddress($stateParams.host, $stateParams.server);
+        return serverGroupService.getCoordinator(server, "clustered");
```

Comment:
This is assuming that the user is using a profile called "clustered", resulting in the page not displaying if they are using a different profile name. We can get the profile name from the serverGroup. So I think it is best to resolve the serverGroup and then resolve the coordinator by passing "serverGroup" as a dependency to the "coord" resolve function.